doc/examples/transform/plot_radon_transform.py

```diff
@@ -98,12 +98,27 @@
 # back projection is among the fastest methods of performing the inverse
 # Radon transform. The only tunable parameter for the FBP is the filter,
 # which is applied to the Fourier transformed projections. It may be used to
-# suppress high frequency noise in the reconstruction. ``skimage`` provides a
-# few different options for the filter.
+# suppress high frequency noise in the reconstruction. ``skimage`` provides
+# the filters 'ramp', 'shepp-logan', 'cosine', 'hamming', and 'hann':
+
+import matplotlib.pyplot as plt
+from skimage.transform.radon_transform import _get_fourier_filter
+
+filters = ["ramp", "shepp-logan", "cosine", "hamming", "hann"]
+for f in filters:
+    response = _get_fourier_filter(2000, f)
+    plt.plot(response, label=f)
+plt.legend()
+plt.xlabel("position in pixel")
+plt.show()
+
+# Applying the inverse radon transformation with the 'ramp' filter can be
+# seen here:
```

Comment:
```suggestion
######################################################################
# Applying the inverse radon transformation with the 'ramp' filter, we get:
```
This will prevent the two figures of this section from getting squished together!